src/main/scala/rocketchip/Periphery.scala

```diff
@@ -17,6 +17,11 @@ import rocket.XLen
 import scala.math.max
 import coreplex._
 
+case object MIFAddrBits extends Field[Int]
+case object MIFDataBits extends Field[Int]
+case object MIFTagBits extends Field[Int]
+case object MIFDataBeats extends Field[Int]
+
```

Comment:
I guess my point was, given we'll need different widths for each exported MMIO port, wouldn't it make sense to just use the .beatBytes field of whatever ParameterKey we have for that port? This is still progress though.